### DISCUSSION TITLE
请问下切换场景有相关的示例吗？

### DISCUSSION THREAD

**Discussion by super-band** [galacean/discussion#32] [Engine] [answered]
目前想要用oasis做一个编辑器，模型导入后摆放好位置，想要保存当前场景。
按照官网给出的基本用法比较简单，只写了如何加载。想知道scene是如何保存为sceneUrl的，请问切换场景有相关的示例或者能给个思路吗？
感谢🙏🙏🙏🙏🙏🙏🙏

```
3. 加载 Scene
如果想要加载 Scene 资产作为应用中的一个场景，可以使用 engine.resourceManager.load 传入 url 即可。

const sceneUrl = "...";

engine.resourceManager.load({ type: AssetType.Scene, url: "..." }).then(scene=>{
  engine.sceneManager.activeScene = scene;
});
```

**Comment by gz65555** [comment, marked as answer]
> scene 如何保存成 scene url

这个是编辑器部分的代码，编辑器暂时还没开源，场景协议也暂时没有开放。目前阶段建议使用 glTF 文件作为场景文件。

> 切换场景有相关的示例或者能给个思路吗

一般是调用一个转场动画，然后调用 `engine.sceneManager.activeScene = newScene` 即可。
